### PR TITLE
ROX-14996 - Add retry in verify syslog test

### PR DESCRIPTION
**Pull request by ksurabhi91**
## Description
Verify syslog notifier is intermittently failing in CI with connection error. Added retry in assert test syslog section to give some time for syslog service to come up after deployment. 

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Ran test few times locally to make sure there are no failures.

### CODE REVIEW

File: qa-tests-backend/src/test/groovy/IntegrationsTest.groovy [hunk 1]
```diff
@@ -184,8 +184,8 @@ class IntegrationsTest extends BaseSpecification {
 
     @Unroll
     @Tag("Integration")
-    // splunk is not supported on P/Z
     @IgnoreIf({ Env.REMOTE_CLUSTER_ARCH == "ppc64le" || Env.REMOTE_CLUSTER_ARCH == "s390x" })
+    // splunk is not supported on P/Z
```
Comment (theencee) [inline qa-tests-backend/src/test/groovy/IntegrationsTest.groovy:188]: why was this moved? Can you please undo?

**Review by clickboo — APPROVED** [review]
(just realized I hadn't published the review!) after addressing Nakul's comment and a comment from me

**Review by theencee — APPROVED** [review]
